revolv/templates/project/edit_project.html

```diff
@@ -132,4 +145,28 @@
         });
     });
 </script>
+<!-- Initializes the category selection tool -->
+<script src="{% static 'chosen/chosen.jquery.js' %}"></script>
+<script src="{% static 'chosen/docsupport/prism.js' %}" type="text/javascript" charset="utf-8"></script>
+<script>
+    $( document ).ready(function() {
+        // populates the currently selected categories if none of them have been selected
```

Comment:
Yeah it's kind of hacky haha, but i think you could at least pass in a list and iterate through it in the template, appending each item to a js array right? that way at least you avoid the string splitting